CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+We hope that you will consider contributing to Fugacious. You can contribute in many ways. For example, you might:
+
+* add documentation and "how-to" articles to the README or Wiki.
+* create new features or new integrations
+* improve and/or add new I18n translations
+* hack on Fugacious itself by fixing bugs you've found in the GitHub Issue tracker or adding new features to Fugacious.
```

Comment:
the second part of the sentence seems redundant to the second bullet, "create new features or new integrations"